Oie docs embedded revisions general 2 (#2244)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

For the sample app use the **Issuer URI** from the custom authorization server
named “default”.  To find this value, go to **Security > API** in the Admin
Console. Under the **Authorization servers** tab, the default authorization

> server’s Issuer URI is displayed in the list.

Reviewer: ```suggestion
Server’s Issuer URI appears in the list.
```

> An example of the URI is:  `https://dev-foo.okta.com/oauth2/default`.

Reviewer: ```suggestion
Example URI:  `https://dev-foo.okta.com/oauth2/default`.
```

> Note that there are two main types of authorization servers: Org and custom:

Reviewer: ```suggestion
> **Note:** There are two main types of authorization servers: Org and Custom
```

> * **Org**: If you want to use the Org authorization server use the following

Reviewer: ```suggestion
* **Org**: If you want to use the Org Authorization Server, use the following
```

>    url: `https://dev-foo.okta.com`.

Reviewer: ```suggestion
   URL: `https://dev-example.okta.com`.
```

> * **Custom**: There are two types of custom authorization servers:

Reviewer: this isn't necessarily accurate. There is the org auth server and then there are custom auth servers. The FIRST custom auth server is automatically created for them. Then, they can create their own additional custom auth servers. Saying that there are two different types of custom auth servers implies that there is a difference between the custom auth server ("default") that is automatically created for them in their org and the ones that they subsequently create after that. I feel that this wording is misleading.

Author: Will improve the clarity of the text in the next PR.
